src/battle/action_cmd.h

```diff
@@ -0,0 +1,175 @@
+    /* 0x4A */ s16 actionCommandID;
+    /* 0x4C */ s16 state;
+    /* 0x4E */ s16 prepareTime;
+    /* 0x50 */ s16 difficulty; // values from 0 to 7
+    /* 0x52 */ s16 duration;
+    /* 0x54 */ s16 frameCounter;
+    /* 0x56 */ s16 hudX;
+    /* 0x58 */ s16 hudY;
+    /* 0x5A */ s16 unk_5A;
+    /* 0x5C */ s8 unk_5C;
+    /* 0x5D */ s8 unk_5D;
+    /* 0x5E */ s8 autoSucceed;
+    /* 0x5F */ s8 hammerMissedStart;
+    /* 0x60 */ s8 wrongButtonPressed;
+    /* 0x61 */ s8 showHud;
+    /* 0x62 */ s8 playHammerSounds;
+    /* 0x63 */ s8 unk_63;
+    /* 0x64 */ s16 targetWeakness; // chance of applying special status, the more the easier it is to fill the bar
+    /* 0x66 */ s16 thresholdMoveDir;
+    /* 0x68 */ s16 isBarFilled;
+    /* 0x6A */ s16 berserkerEnabled;
+    /* 0x6C */ s16 hudPrepareTime;
+    /* 0x6E */ s16 hitsTakenIsMax;
+    /* 0x70 */ s16 lookBackCounter;
+    /* 0x72 */ s16 wrongInputFrameCounter;
+    /* 0x74 */ s16 mashMeterCutoffs[6]; // upper bounds for each interval
+    /* 0x80 */ s8 mashMeterIntervals;
+} ActionCommandStatus;
+
+extern ActionCommandStatus gActionCommandStatus; // TODO BSS?
+
+extern s32 actionCmdTableJump[];
+extern s32 actionCmdTableHammer[];
+extern s32 actionCmdTableFlee[];
+extern s32 actionCmdTableBreakFree[];
+extern s32 actionCmdTableWhirlwind[];
+extern s32 actionCmdTableStopLeech[];
+extern s32 actionCmdTable07[];
+extern s32 actionCmdTableDizzyShell[];
+extern s32 actionCmdTableFireShell[];
+extern s32 actionCmdTable0A[];
+extern s32 actionCmdTableBomb[];
+extern s32 actionCmdTableBodySlam[];
+extern s32 actionCmdTableAirLift[];
+extern s32 actionCmdTableAirRaid[];
+extern s32 actionCmdTableSquirt[];
+extern s32 actionCmdTablePowerShock[];
+extern s32 actionCmdTableMegaShock[];
+extern s32 actionCmdTableSmack[];
+extern s32 actionCmdTableSpinySurge[];
+extern s32 actionCmdTableHurricane[];
+extern s32 actionCmdTableSpook[];
+extern s32 actionCmdTableWaterBlock[];
+extern s32 actionCmdTableTidalWave[];
+
+extern HudScript HES_BlueMeter;
+extern HudScript HES_BButton;
+extern HudScript HES_MashBButton1;
+extern HudScript HES_AButton;
+extern HudScript HES_RunningAway;
+extern HudScript HES_RunAwayOK;
+extern HudScript HES_100pct;
+extern HudScript HES_FillGaugeResult;
+extern HudScript HES_MashAButton;
+extern HudScript HES_StickHoldLeft;
+extern HudScript HES_StickTapNeutral;
+extern HudScript HES_StickNeutral;
+extern HudScript HES_StickMashLeft;
+extern HudScript HES_TimingWait;
+extern HudScript HES_RightOn;
+extern HudScript HES_AButtonDown;
+extern HudScript HES_TimingReady;
+extern HudScript HES_TimingMiss;
+extern HudScript HES_TimingOK;
+extern HudScript HES_TimingBar1Chance;
+extern HudScript HES_TimingBar3Chances;
+extern HudScript HES_TimingCharge1;
+extern HudScript HES_TimingCharge2;
+extern HudScript HES_TimingCharge3;
+extern HudScript HES_TimingCharge4a;
+extern HudScript HES_TimingCharge4b;
+extern HudScript HES_TimingCharge4c;
+extern HudScript HES_PressAButton;
```

Comment:
HES_PressAButton is already declared line 108

Reply:
it's HES_AButton on line 108